README.md

```diff
@@ -68,7 +68,7 @@ Organizations below are **officially** using Argo CD. Please send a PR with your
 1. [Viaduct](https://www.viaduct.ai/)
 1. [Volvo Cars](https://www.volvocars.com/)
 1. [Walkbase](https://www.walkbase.com/)
-
+1. [Whitehat Berlin](https://whitehat.berlin) at [ATU](https://www.atu.de/)/[Mobivia](https://www.mobivia.com/) & [UMI](https://www.linkedin.com/company/urban-mobility-international/)/[WeShare](http://we-share.io/)/[Volkswagen](http://volkswagen.de/)
```

Comment:
Just a question, because I know the corporate dos-and-donts too well: Do you have permission to use these references?

Reply:
@jannfis I just sent a CYA mail to both... 
clarifying that this is a "Thank you UMI & ATU for sponsoring my work on ArgoCD" and nothing else

Reply:
just sync up with @edlee2121 about it. We think that this list should not have references. 
They might add themself if they want :) 

@zeph , could you please remove the references and keep just the company name?